Algebraic_foundations/include/CGAL/number_utils.h

```diff
@@ -107,7 +107,7 @@ sqrt( const AS& x ) {
 template <typename NT>
 NT approximate_sqrt(const NT& nt, CGAL::Field_tag)
 {
-  return NT(sqrt(CGAL::to_double(nt)));
+  return NT(sqrt(to_double(nt)));
```

Comment:
If `NT` is `double`, then the call `to_double(nt)` will not compile.

The solution is to qualify the call: `CGAL::to_double`, but this definition of `approximate_sqrt` should be move to after the line 320, where `CGAL::to_double` is declared and defined.